tutorials/nemo-retriever-synthetic-data-generation/retriever_evalset_generator.py

```diff
@@ -0,0 +1,187 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import hashlib
+import importlib
+import os
+import re
+import secrets
+from abc import ABC, abstractmethod
+from typing import Any
+
+import dask.array as da
+import dask.dataframe as dd
+import pandas as pd
+from dask.base import normalize_token, tokenize
+from dask.diagnostics import ProgressBar
+from dask.distributed import progress
+from distributed import Client
+from omegaconf import DictConfig, OmegaConf
+from openai import AsyncOpenAI, OpenAI
+from tqdm import tqdm
+from tqdm.dask import TqdmCallback
+
+from nemo_curator import AsyncOpenAIClient, OpenAIClient
+from nemo_curator.datasets import DocumentDataset
+from nemo_curator.filters.doc_filter import DocumentFilter
+from nemo_curator.synthetic import AsyncNemotronGenerator, NemotronGenerator
+from nemo_curator.synthetic.generator import SyntheticDataGenerator
+
+config = importlib.import_module(
```

Comment:
Does the following import not work?
```python
from config import config
```

`tutorials/peft-curation` does something similar [here](https://github.com/NVIDIA/NeMo-Curator/blob/main/tutorials/peft-curation/main.py#L21).

Reply:
fixed this!